From 8a1e25d35c17823c5ba102f54e9316c56d545b4d Mon Sep 17 00:00:00 2001
From: KoysX <2685697179b@gmail.com>
Date: Wed, 25 Oct 2023 10:42:25 -0400
Subject: [PATCH] Replace-build-name

---
 config/version.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/version.mk b/config/version.mk
index 8072a21..563bcc7 100644
--- a/config/version.mk
+++ b/config/version.mk
@@ -25,7 +25,7 @@ TARGET_PRODUCT_SHORT := $(subst derp_,,$(DERP_BUILDTYPE))
 DERP_BUILDTYPE ?= Community
 DERP_STATUS := Alpha
 DERP_BUILD_VERSION := $(PLATFORM_VERSION)
-DERP_VERSION := $(DERP_BUILD_VERSION)-$(DERP_BUILDTYPE)-$(DERP_STATUS)-$(DERP_BUILD)-$(BUILD_DATE)
+DERP_VERSION := $(DERP_BUILD_VERSION)-$(DERP_BUILDTYPE)-$(DERP_STATUS)-gsi-$(DERP_BUILD)-$(BUILD_DATE)
 ROM_FINGERPRINT := DerpFest/$(PLATFORM_VERSION)/$(TARGET_PRODUCT_SHORT)/$(shell date -u +%H%M)
 
 ifeq ($(DERP_BUILDTYPE), CI)
-- 
2.42.0

