From 4a0069e8f663f369bb9c1044bc32a9dde48c5585 Mon Sep 17 00:00:00 2001
From: BoNic <BoNio9527@gmaul.com>
Date: Sun, 17 Dec 2023 12:43:45 +0800
Subject: [PATCH] add-omnijaws

---
 config/packages.mk | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/config/packages.mk b/config/packages.mk
index 2124ba1..cb2bf2e 100644
--- a/config/packages.mk
+++ b/config/packages.mk
@@ -7,7 +7,8 @@ PRODUCT_PACKAGES += \
     Panic \
     Ripple \
     TouchGestures \
-    Updater
+    Updater \
+    OmniJaws
 
 # Themes
 PRODUCT_PACKAGES += \
-- 
2.34.1

