From 298e09247e4587a3d9417c14a27a41b25315427d Mon Sep 17 00:00:00 2001
From: BoNic <BoNio9527@gmail.com>
Date: Tue, 7 Nov 2023 15:22:41 +0800
Subject: [PATCH] Every-voldmanaged-storage-is-adoptable

---
 main.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/main.cpp b/main.cpp
index e299476..d19f051 100644
--- a/main.cpp
+++ b/main.cpp
@@ -275,7 +275,7 @@ static int process_config(VolumeManager* vm, VoldConfigs* configs) {
             int partnum = entry.partnum;
             int flags = 0;
 
-            if (entry.is_encryptable()) {
+            if (entry.is_encryptable() || true) {
                 flags |= android::vold::Disk::Flags::kAdoptable;
                 configs->has_adoptable = true;
             }
-- 
2.34.1

