From 0d360d7d92956b37bcd42f0bf1140dccb1a65b98 Mon Sep 17 00:00:00 2001
From: KoysX <2685697179b@gmail.com>
Date: Mon, 7 Aug 2023 09:32:57 +0800
Subject: [PATCH] Force-treble-apk-as-the-default-updater

---
 .../settings/system/SystemUpdatePreferenceController.java       | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/com/android/settings/system/SystemUpdatePreferenceController.java b/src/com/android/settings/system/SystemUpdatePreferenceController.java
index 3b91e90..b906195 100644
--- a/src/com/android/settings/system/SystemUpdatePreferenceController.java
+++ b/src/com/android/settings/system/SystemUpdatePreferenceController.java
@@ -43,6 +43,8 @@ public class SystemUpdatePreferenceController extends BasePreferenceController {
 
     private static final String TAG = "SysUpdatePrefContr";
 
+    private static final String OTA_APP_PACKAGE = "me.phh.treble.app";
+
     private static final String KEY_SYSTEM_UPDATE_SETTINGS = "system_update_settings";
 
     private final UserManager mUm;
-- 
2.34.1

