From a846a93e47595defd4a406bc04ca6b13a915faca Mon Sep 17 00:00:00 2001
From: BoNic <BoNio9527@gmaul.com>
Date: Sun, 17 Dec 2023 11:38:32 +0800
Subject: [PATCH] Use-App-icon-for-per-app-volume

---
 .../systemui/volume/VolumeDialogImpl.java     | 23 ++++++++++++++++++-
 1 file changed, 22 insertions(+), 1 deletion(-)

diff --git a/packages/SystemUI/src/com/android/systemui/volume/VolumeDialogImpl.java b/packages/SystemUI/src/com/android/systemui/volume/VolumeDialogImpl.java
index 9c0b1f379..72ac4b7ea 100644
--- a/packages/SystemUI/src/com/android/systemui/volume/VolumeDialogImpl.java
+++ b/packages/SystemUI/src/com/android/systemui/volume/VolumeDialogImpl.java
@@ -273,6 +273,7 @@ public class VolumeDialogImpl implements VolumeDialog, Dumpable,
     private FrameLayout mZenIcon;
     private View mAppVolumeView;
     private ImageButton mAppVolumeIcon;
+    private String mAppVolumeActivePackageName;
     private final List<VolumeRow> mRows = new ArrayList<>();
     private ConfigurableTexts mConfigurableTexts;
     private final SparseBooleanArray mDynamic = new SparseBooleanArray();
@@ -1430,19 +1431,32 @@ public class VolumeDialogImpl implements VolumeDialog, Dumpable,
         ContentResolver cr = mContext.getContentResolver();
         int showAppVolume = Settings.System.getInt(cr, Settings.System.SHOW_APP_VOLUME, 0);
         boolean ret = showAppVolume == 1;
+        mAppVolumeActivePackageName = null;
         if (ret) {
             ret = false;
             AudioManager audioManager = mController.getAudioManager();
             for (AppVolume av : audioManager.listAppVolumes()) {
                 if (av.isActive()) {
                     ret = true;
-            break;
+                    mAppVolumeActivePackageName = av.getPackageName();
+                    break;
                 }
             }
         }
         return ret;
     }
 
+    private Drawable getApplicationIcon(String packageName) {
+        PackageManager pm = mContext.getPackageManager();
+        Drawable icon = null;
+        try {
+            icon = pm.getApplicationIcon(packageName);
+        } catch (Exception e) {
+            // nothing to do
+        }
+        return icon;
+    }
+
     public void initAppVolumeH() {
         if (mAppVolumeView != null) {
             mAppVolumeView.setVisibility(shouldShowAppVolume() ? VISIBLE : GONE);
@@ -1456,6 +1470,13 @@ public class VolumeDialogImpl implements VolumeDialog, Dumpable,
                 Dependency.get(ActivityStarter.class).startActivity(intent,
                         true /* dismissShade */);
             });
+            Drawable icon = mAppVolumeActivePackageName != null ?
+                    getApplicationIcon(mAppVolumeActivePackageName) : null;
+            if (icon != null) {
+                mAppVolumeIcon.setImageTintList(null);
+                mAppVolumeIcon.setScaleType(ImageView.ScaleType.FIT_CENTER);
+                mAppVolumeIcon.setImageDrawable(icon);
+            }
         }
     }
 
-- 
2.34.1

