From 1d822515b63c901cf72da68fb4cfb73080e3b1ac Mon Sep 17 00:00:00 2001
From: BoNic <BoNio9527@gmaul.com>
Date: Thu, 28 Dec 2023 11:05:34 +0800
Subject: [PATCH] slim

---
 pixel-vendor.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/pixel-vendor.mk b/pixel-vendor.mk
index 35035c7..c43a18c 100644
--- a/pixel-vendor.mk
+++ b/pixel-vendor.mk
@@ -96,7 +96,6 @@ PRODUCT_COPY_FILES += \
 PRODUCT_PACKAGES += \
     MarkupGoogle \
     SoundPickerPrebuilt \
-    AndroidAutoStubPrebuilt \
     DeviceIntelligenceNetworkPrebuilt \
     DevicePersonalizationPrebuiltPixel2020 \
     SettingsIntelligenceGooglePrebuilt \
-- 
2.34.1

