From 648bd83bdac6377f0c6bc58ca1130d9f5d5383bd Mon Sep 17 00:00:00 2001
From: KoysX <2685697179b@gmail.com>
Date: Tue, 24 Oct 2023 23:24:43 -0400
Subject: [PATCH] init-Do-not-start-console-service-when-debuggable

---
 rootdir/init.rc | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/rootdir/init.rc b/rootdir/init.rc
index 84589e3..e2368cc 100644
--- a/rootdir/init.rc
+++ b/rootdir/init.rc
@@ -1310,9 +1310,6 @@ on property:ro.debuggable=1
     # Give reads to anyone for the accessibility trace folder on debug builds.
     chmod 0775 /data/misc/a11ytrace
 
-on init && property:ro.debuggable=1 && property:ro.console.enable=1
-    start console
-
 on userspace-reboot-requested
   # TODO(b/135984674): reset all necessary properties here.
   setprop sys.boot_completed ""
-- 
2.42.0

