From 727bdcdb393afaf1f0f6264e973e8bf2937c65d8 Mon Sep 17 00:00:00 2001
From: KoysX <2685697179b@gmail.com>
Date: Tue, 22 Aug 2023 18:56:22 +0800
Subject: [PATCH] Revert-Set-system-xbin-permissions-to-750

---
 libcutils/fs_config.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libcutils/fs_config.cpp b/libcutils/fs_config.cpp
index 6f7cbff..0d9736a 100644
--- a/libcutils/fs_config.cpp
+++ b/libcutils/fs_config.cpp
@@ -88,7 +88,7 @@ static const struct fs_path_config android_dirs[] = {
     { 00755, AID_ROOT,         AID_SHELL,        0, "system/etc" },
     { 00755, AID_ROOT,         AID_ROOT,         0, "system/etc/ppp" },
     { 00755, AID_ROOT,         AID_SHELL,        0, "system/vendor" },
-    { 00750, AID_ROOT,         AID_SHELL,        0, "system/xbin" },
+    { 00751, AID_ROOT,         AID_SHELL,        0, "system/xbin" },
     { 00751, AID_ROOT,         AID_SHELL,        0, "system/apex/*/bin" },
     { 00751, AID_ROOT,         AID_SHELL,        0, "system_ext/bin" },
     { 00751, AID_ROOT,         AID_SHELL,        0, "system_ext/apex/*/bin" },
-- 
2.34.1

