From 57641b1f1a68aa0bb69f4221c74273178d49ee0e Mon Sep 17 00:00:00 2001
From: KoysX <2685697179b@gmail.com>
Date: Wed, 25 Oct 2023 10:46:54 -0400
Subject: [PATCH] slim

---
 common/common-vendor.mk | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/common/common-vendor.mk b/common/common-vendor.mk
index 105391e..3680a01 100644
--- a/common/common-vendor.mk
+++ b/common/common-vendor.mk
@@ -6,7 +6,6 @@ PRODUCT_SOONG_NAMESPACES += \
     vendor/gms/common
 
 PRODUCT_COPY_FILES += \
-    vendor/gms/common/proprietary/product/app/Chrome/Chrome.apk.gz:$(TARGET_COPY_OUT_PRODUCT)/app/Chrome/Chrome.apk.gz \
     vendor/gms/common/proprietary/product/app/TrichromeLibrary/TrichromeLibrary.apk.gz:$(TARGET_COPY_OUT_PRODUCT)/app/TrichromeLibrary/TrichromeLibrary.apk.gz \
     vendor/gms/common/proprietary/product/app/WebViewGoogle/WebViewGoogle.apk.gz:$(TARGET_COPY_OUT_PRODUCT)/app/WebViewGoogle/WebViewGoogle.apk.gz \
     vendor/gms/common/proprietary/product/etc/default-permissions/default-permissions.xml:$(TARGET_COPY_OUT_PRODUCT)/etc/default-permissions/default-permissions.xml \
@@ -89,13 +88,11 @@ PRODUCT_COPY_FILES += \
 PRODUCT_PACKAGES += \
     GoogleExtShared \
     GooglePrintRecommendationService \
-    DocumentsUIGoogle \
     GoogleExtServices \
     GooglePackageInstaller \
     TagGoogle \
     CalculatorGooglePrebuilt \
     CalendarGooglePrebuilt \
-    Chrome-Stub \
     DevicePolicyPrebuilt \
     GoogleContacts \
     GoogleTTS \
@@ -109,19 +106,14 @@ PRODUCT_PACKAGES += \
     PixelThemesStub \
     PixelWallpapers2020 \
     PrebuiltDeskClockGoogle \
-    PrebuiltGmail \
     SoundAmplifierPrebuilt \
     SoundPickerPrebuilt \
     TrichromeLibrary-Stub \
     WebViewGoogle-Stub \
-    YouTube \
     arcore \
-    talkback \
-    AndroidAutoStubPrebuilt \
     ConfigUpdater \
     DeviceIntelligenceNetworkPrebuilt \
     DevicePersonalizationPrebuiltPixel2020 \
-    FilesPrebuilt \
     GoogleDialer \
     GoogleOneTimeInitializer \
     GoogleRestorePrebuilt \
@@ -140,7 +132,6 @@ PRODUCT_PACKAGES += \
     PrebuiltGmsCoreSc_MeasurementDynamite \
     AndroidPlatformServices \
     RecorderPrebuilt \
-    SafetyHubPrebuilt \
     ScribePrebuilt \
     SettingsIntelligenceGooglePrebuilt \
     SetupWizardPrebuilt \
@@ -150,7 +141,6 @@ PRODUCT_PACKAGES += \
     EmergencyInfoGoogleNoUi \
     Flipendo \
     GoogleServicesFramework \
-    NexusLauncherRelease \
     PixelSetupWizard \
     StorageManagerGoogle \
     SystemUIClocks-BigNum \
-- 
2.42.0

