From 61e3b40f68895985615056f76107bc8b57ece262 Mon Sep 17 00:00:00 2001
From: KoysX <2685697179b@gmail.com>
Date: Mon, 31 Jul 2023 13:08:41 +0800
Subject: [PATCH] slim

---
 common/common-vendor.mk | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/common/common-vendor.mk b/common/common-vendor.mk
index 5d0c1b6..04d2867 100644
--- a/common/common-vendor.mk
+++ b/common/common-vendor.mk
@@ -6,7 +6,6 @@ PRODUCT_SOONG_NAMESPACES += \
     vendor/gms/common
 
 PRODUCT_COPY_FILES += \
-    vendor/gms/common/proprietary/product/app/Chrome/Chrome.apk.gz:$(TARGET_COPY_OUT_PRODUCT)/app/Chrome/Chrome.apk.gz \
     vendor/gms/common/proprietary/product/app/TrichromeLibrary/TrichromeLibrary.apk.gz:$(TARGET_COPY_OUT_PRODUCT)/app/TrichromeLibrary/TrichromeLibrary.apk.gz \
     vendor/gms/common/proprietary/product/app/WebViewGoogle/WebViewGoogle.apk.gz:$(TARGET_COPY_OUT_PRODUCT)/app/WebViewGoogle/WebViewGoogle.apk.gz \
     vendor/gms/common/proprietary/product/etc/default-permissions/default-permissions.xml:$(TARGET_COPY_OUT_PRODUCT)/etc/default-permissions/default-permissions.xml \
@@ -33,12 +32,9 @@ PRODUCT_COPY_FILES += \
 PRODUCT_PACKAGES += \
     GoogleExtShared \
     GooglePrintRecommendationService \
-    DocumentsUIGoogle \
-    GooglePackageInstaller \
     TagGoogle \
     CalculatorGooglePrebuilt \
     CalendarGooglePrebuilt \
-    Chrome-Stub \
     GoogleContacts \
     GoogleTTS \
     LatinIMEGooglePrebuilt \
@@ -50,12 +46,9 @@ PRODUCT_PACKAGES += \
     SoundPickerPrebuilt \
     TrichromeLibrary-Stub \
     WebViewGoogle-Stub \
-    talkback \
-    AndroidAutoStubPrebuilt \
     ConfigUpdater \
     DeviceIntelligenceNetworkPrebuilt \
     DevicePersonalizationPrebuiltPixel2020 \
-    FilesPrebuilt \
     GoogleDialer \
     GoogleOneTimeInitializer \
     GoogleRestorePrebuilt \
@@ -64,7 +57,6 @@ PRODUCT_PACKAGES += \
     PrebuiltBugle \
     PrebuiltGmsCoreSc \
     AndroidPlatformServices \
-    SafetyHubPrebuilt \
     ScribePrebuilt \
     SecurityHubPrebuilt \
     SettingsIntelligenceGooglePrebuilt \
-- 
2.34.1

