From c61e0587074ad5657aad24502e30a17533e5b977 Mon Sep 17 00:00:00 2001
From: KoysX <2685697179b@gmail.com>
Date: Thu, 20 Jul 2023 09:03:13 +0800
Subject: [PATCH] Revert-udfps-Forward-port-Boost-framework-hook

---
 .../systemui/biometrics/UdfpsController.java  | 19 -------------------
 1 file changed, 19 deletions(-)

diff --git a/packages/SystemUI/src/com/android/systemui/biometrics/UdfpsController.java b/packages/SystemUI/src/com/android/systemui/biometrics/UdfpsController.java
index 26223103..7fa587f6 100644
--- a/packages/SystemUI/src/com/android/systemui/biometrics/UdfpsController.java
+++ b/packages/SystemUI/src/com/android/systemui/biometrics/UdfpsController.java
@@ -52,7 +52,6 @@ import android.os.VibrationAttributes;
 import android.os.VibrationEffect;
 import android.os.UserHandle;
 import android.provider.Settings;
-import android.util.BoostFramework;
 import android.util.Log;
 import android.util.RotationUtils;
 import android.view.LayoutInflater;
@@ -210,11 +209,6 @@ public class UdfpsController implements DozeReceiver, Dumpable {
 
     private UdfpsAnimation mUdfpsAnimation;
 
-    // Boostframework for UDFPS
-    private BoostFramework mPerf = null;
-    private boolean mIsPerfLockAcquired = false;
-    private static final int BOOST_DURATION_TIMEOUT = 2000;
-
     @VisibleForTesting
     public static final VibrationAttributes UDFPS_VIBRATION_ATTRIBUTES =
             new VibrationAttributes.Builder()
@@ -838,7 +832,6 @@ public class UdfpsController implements DozeReceiver, Dumpable {
         udfpsShell.setUdfpsOverlayController(mUdfpsOverlayController);
 
         mUdfpsVendorCode = mContext.getResources().getInteger(R.integer.config_udfps_vendor_code);
-        mPerf = new BoostFramework();
 
         mAmbientDisplayConfiguration = new AmbientDisplayConfiguration(mContext);
         updateScreenOffFodState();
@@ -932,13 +925,6 @@ public class UdfpsController implements DozeReceiver, Dumpable {
         } else {
             Log.v(TAG, "showUdfpsOverlay | the overlay is already showing");
         }
-
-        if (mPerf != null && !mIsPerfLockAcquired) {
-            mPerf.perfHint(BoostFramework.VENDOR_HINT_FIRST_LAUNCH_BOOST,
-                    null,
-                    BOOST_DURATION_TIMEOUT);
-            mIsPerfLockAcquired = true;
-        }
     }
 
     private void hideUdfpsOverlay() {
@@ -1229,11 +1215,6 @@ public class UdfpsController implements DozeReceiver, Dumpable {
             unconfigureDisplay(view);
         }
         cancelAodSendFingerUpAction();
-
-            if (mPerf != null && mIsPerfLockAcquired) {
-            mPerf.perfLockRelease();
-            mIsPerfLockAcquired = false;
-        }
     }
 
     /**
-- 
2.34.1

