From fa3690ebe05b7a6ff452127165e8036b4fb5f2cc Mon Sep 17 00:00:00 2001
From: KoysX <2685697179b@gmail.com>
Date: Thu, 20 Jul 2023 10:42:43 +0800
Subject: [PATCH] slim-app

---
 target/product/handheld_product.mk | 2 --
 1 file changed, 2 deletions(-)

diff --git a/target/product/handheld_product.mk b/target/product/handheld_product.mk
index 2dd1d7f..73ff3d4 100644
--- a/target/product/handheld_product.mk
+++ b/target/product/handheld_product.mk
@@ -22,10 +22,8 @@ $(call inherit-product, $(SRC_TARGET_DIR)/product/media_product.mk)
 
 # /product packages
 PRODUCT_PACKAGES += \
-    Calendar \
     Contacts \
     DeskClock \
-    Gallery2 \
     Music \
     OneTimeInitializer \
     preinstalled-packages-platform-handheld-product.xml \
-- 
2.34.1

