From 2eba9bb671eed01e707d147dd56b20f344fcfe5f Mon Sep 17 00:00:00 2001
From: BoNic <BoNio9527@gmail.com>
Date: Tue, 5 Dec 2023 04:38:29 -0500
Subject: [PATCH] Remove smartspace view when OmniJaws weather is enabled

---
 .../KeyguardClockSwitchController.java        | 74 +++++++++----------
 1 file changed, 37 insertions(+), 37 deletions(-)

diff --git a/packages/SystemUI/src/com/android/keyguard/KeyguardClockSwitchController.java b/packages/SystemUI/src/com/android/keyguard/KeyguardClockSwitchController.java
index 717e5a3cb..e2631b070 100644
--- a/packages/SystemUI/src/com/android/keyguard/KeyguardClockSwitchController.java
+++ b/packages/SystemUI/src/com/android/keyguard/KeyguardClockSwitchController.java
@@ -120,7 +120,9 @@ public class KeyguardClockSwitchController extends ViewController<KeyguardClockS
     private final ContentObserver mShowWeatherObserver = new ContentObserver(null) {
         @Override
         public void onChange(boolean change) {
-            setWeatherVisibility();
+            if (!mShowWeather) {
+                setWeatherVisibility();
+            }
         }
     };

@@ -204,8 +206,6 @@ public class KeyguardClockSwitchController extends ViewController<KeyguardClockS

         mDumpManager.unregisterDumpable(getClass().toString()); // unregister previous clocks
         mDumpManager.registerDumpable(getClass().toString(), this);
-
-        mTunerService.addTunable(this, LOCKSCREEN_WEATHER_ENABLED);
     }

     @Override
@@ -219,7 +219,6 @@ public class KeyguardClockSwitchController extends ViewController<KeyguardClockS
                 mView.getResources().getDimensionPixelSize(R.dimen.keyguard_large_clock_top_margin);
         mKeyguardDateWeatherViewInvisibility =
                 mView.getResources().getInteger(R.integer.keyguard_date_weather_view_invisibility);
-        updateWeatherView();

         if (mOnlyClock) {
             View ksv = mView.findViewById(R.id.keyguard_slice_view);
@@ -234,20 +233,9 @@ public class KeyguardClockSwitchController extends ViewController<KeyguardClockS

         mStatusArea = mView.findViewById(R.id.keyguard_status_area);

-        if (mSmartspaceController.isEnabled()) {
-            View ksv = mView.findViewById(R.id.keyguard_slice_view);
-            int viewIndex = mStatusArea.indexOfChild(ksv);
-            ksv.setVisibility(View.GONE);
+        mTunerService.addTunable(this, LOCKSCREEN_WEATHER_ENABLED);

-            // TODO(b/261757708): add content observer for the Settings toggle and add/remove
-            //  weather according to the Settings.
-            if (mSmartspaceController.isDateWeatherDecoupled()) {
-                addDateWeatherView(viewIndex);
-                viewIndex += 1;
-            }
-
-            addSmartspaceView(viewIndex);
-        }
+        updateViews();

         mSecureSettings.registerContentObserverForUser(
                 Settings.Secure.LOCKSCREEN_USE_DOUBLE_LINE_CLOCK,
@@ -264,13 +252,40 @@ public class KeyguardClockSwitchController extends ViewController<KeyguardClockS
         );

         updateDoubleLineClock();
-        setDateWeatherVisibility();
-        setWeatherVisibility();

         mKeyguardUnlockAnimationController.addKeyguardUnlockAnimationListener(
                 mKeyguardUnlockAnimationListener);
     }

+    private void updateViews() {
+        if (mSmartspaceController.isEnabled()) {
+            View ksv = mView.findViewById(R.id.keyguard_slice_view);
+            int viewIndex = mStatusArea.indexOfChild(ksv);
+            ksv.setVisibility(mShowWeather ? View.VISIBLE : View.GONE);
+
+            if (mSmartspaceController.isDateWeatherDecoupled()) {
+                int index = mStatusArea.indexOfChild(mDateWeatherView);
+                if (index >= 0) {
+                    mDateWeatherView.removeView(mWeatherView);
+                    mStatusArea.removeView(mDateWeatherView);
+                }
+                if (!mShowWeather) {
+                    addDateWeatherView(viewIndex);
+                    setDateWeatherVisibility();
+                    setWeatherVisibility();
+                    viewIndex += 1;
+                }
+            }
+            int index = mStatusArea.indexOfChild(mSmartspaceView);
+            if (index >= 0) {
+                mStatusArea.removeView(mSmartspaceView);
+            }
+            if (!mShowWeather) {
+                addSmartspaceView(viewIndex);
+            }
+        }
+    }
+
     int getNotificationIconAreaHeight() {
         return mNotificationIconAreaController.getHeight();
     }
@@ -318,26 +333,11 @@ public class KeyguardClockSwitchController extends ViewController<KeyguardClockS
                 }
             }
         });
+        updateViews();
     }

     void onLocaleListChanged() {
-        if (mSmartspaceController.isEnabled()) {
-            if (mSmartspaceController.isDateWeatherDecoupled()) {
-                mDateWeatherView.removeView(mWeatherView);
-                int index = mStatusArea.indexOfChild(mDateWeatherView);
-                if (index >= 0) {
-                    mStatusArea.removeView(mDateWeatherView);
-                    addDateWeatherView(index);
-                }
-                setDateWeatherVisibility();
-                setWeatherVisibility();
-            }
-            int index = mStatusArea.indexOfChild(mSmartspaceView);
-            if (index >= 0) {
-                mStatusArea.removeView(mSmartspaceView);
-                addSmartspaceView(index);
-            }
-        }
+        updateViews();
     }

     private void addDateWeatherView(int index) {
@@ -615,6 +615,6 @@ public class KeyguardClockSwitchController extends ViewController<KeyguardClockS
         }

         return ((mCurrentClockSize == LARGE) ? clock.getLargeClock() : clock.getSmallClock())
-                .getConfig().getHasCustomWeatherDataDisplay();
+                .getConfig().getHasCustomWeatherDataDisplay() && !mShowWeather;
     }
 }
--
2.25.1

