From 888f46b04e70015afba39c8cc7ae3dd8291a986f Mon Sep 17 00:00:00 2001
From: KoysX <2685697179b@gmail.com>
Date: Tue, 5 Sep 2023 08:12:33 +0800
Subject: [PATCH] treble-Force-enable-Wireless-Display

---
 system.prop | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/system.prop b/system.prop
index 986cb09..5fe3d9e 100644
--- a/system.prop
+++ b/system.prop
@@ -53,3 +53,6 @@ vendor.usb.use_ffs_mtp=1
 # Override media volume steps
 ro.config.media_vol_steps=25
 ro.config.media_vol_default=8
+
+# Wireless Display
+persist.debug.wfd.enable=1
-- 
2.34.1

