From d48be23189280f15e4071e8a393951509fc24e22 Mon Sep 17 00:00:00 2001
From: KoysX <2685697179b@gmail.com>
Date: Sun, 6 Aug 2023 18:50:30 +0800
Subject: [PATCH] Add-a-property-to-cap-declared-le-vendor-version.-Fo

---
 system/gd/hci/controller.cc | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/system/gd/hci/controller.cc b/system/gd/hci/controller.cc
index 332745d..a3145cd 100644
--- a/system/gd/hci/controller.cc
+++ b/system/gd/hci/controller.cc
@@ -514,6 +514,13 @@ struct Controller::impl {
     if (complete_view.IsValid()) {
       vendor_capabilities_.is_supported_ = 0x01;
 
+      int vendor_cap_max = 0xffff;
+      std::string vendor_cap_max_prop = GetSystemProperty("persist.sys.bt.max_vendor_cap").value_or("");
+      if (vendor_cap_max_prop != "") {
+          vendor_cap_max = std::stoi(vendor_cap_max_prop);
+      }
+      if (vendor_cap_max < 55) return;
+
       // v0.55
       BaseVendorCapabilities base_vendor_capabilities = complete_view.GetBaseVendorCapabilities();
       vendor_capabilities_.max_advt_instances_ = base_vendor_capabilities.max_advt_instances_;
@@ -529,6 +536,8 @@ struct Controller::impl {
         return;
       }
 
+      if (vendor_cap_max < 95) return;
+
       // v0.95
       auto v95 = LeGetVendorCapabilitiesComplete095View::Create(complete_view);
       if (!v95.IsValid()) {
@@ -543,6 +552,7 @@ struct Controller::impl {
         return;
       }
 
+      if (vendor_cap_max < 96) return;
       // v0.96
       auto v96 = LeGetVendorCapabilitiesComplete096View::Create(v95);
       if (!v96.IsValid()) {
@@ -554,6 +564,7 @@ struct Controller::impl {
         return;
       }
 
+      if (vendor_cap_max < 98) return;
       // v0.98
       auto v98 = LeGetVendorCapabilitiesComplete098View::Create(v96);
       if (!v98.IsValid()) {
-- 
2.34.1

